### PR TITLE
done with landing page

### PR DESCRIPTION
**Pull request by Varun-28**
Done with the landing page.

- Added navigation
- Updated Navbar with login button and search field
- made everything responsive

Screenshot 2022-04-14 at 07-33-53 CraftTube

Screenshot 2022-04-14 at 07-34-09 CraftTube

### CODE REVIEW

**Review by RakshithBhat03 — APPROVED** [review]
- The code looks clean and well structured.
- Good use of utility classes & css variables
- Good folder structured followed

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
+          style={{ fontFamily: "Arial, FontAwesome" }}
+          placeholder="&#xF002;"
+        />
+      </div>
+      <div className="nav-links">
+        <Link to="/login" className="mx-4">
+          <button
+            className="btn btn-primary"
+            onClick={themeHandler}
+            style={{
```
Comment (abhij1607) [inline src/components/Navbar.jsx]: suggestion to avoid inline style, try to do by changing class

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -1,27 +1,50 @@
...
+              backgroundColor: theme.mode.textColor,
+            }}
+          >
+            Login
+          </button>
+        </Link>
+        <button
+          className="btn-theme-change"
+          onClick={themeHandler}
+          style={{ color: theme.mode.textColor }}
```
Comment (abhij1607) [inline src/components/Navbar.jsx]: avoid inline style

**Review by abhij1607 — COMMENTED** [review]
Neat & Clean code | | good naming used | | left a suggestion in comments

File: src/components/Navbar.jsx (hunk 1 above)
Comment (Varun-28) [reply src/components/Navbar.jsx]: done changes as you suggested

File: src/components/Navbar.jsx (hunk 2 above)
Comment (Varun-28) [reply src/components/Navbar.jsx]: done changes as you suggested

**Pull request merged**